packages/world/src/systems/admin/OracleSystem.sol

```diff
@@ -10,14 +10,14 @@ import { Vec3 } from "../../Vec3.sol";
 import { PHYSICS_ORACLE_ADDRESS } from "../../Constants.sol";
 
 contract OracleSystem is System {
-  function oracleSetInitialEnergyPool(Vec3 shardCoord, uint128 energy) public {
+  function oracleSetInitialEnergyPool(Vec3 fragmentCoord, uint128 energy) public {
```

Comment:
Energy pools are per shard, cuz they're 2d, fragment is 3d so this shouldnt be renamed